### PR TITLE
V0 Misumi Led Mount 

### PR DESCRIPTION
**Pull request by GearNut**
* [ X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ X] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [ X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X ] Submitted STL files are oriented and scaled properly for printing.
  * [ X] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [ X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Adding a new led mount for the V0 that works with either the stock top hat or feasibly any similar design and shape to the tophat area with minimal printed parts and cost 

#### How was it tested? 
It was personally designed and tested by myself and actively in use in my V0

#### Any background context you want to provide?
The step files were the STL loaded into fusion 360 and exported as STEP Im a hobby designer so no idea if those work for what others may need them for

#### Screenshots (if appropriate)
N/A

#### Further notes
Learning github is complicated.

### CODE REVIEW

**Comment by newphreak** [comment]
.step should be in a /CAD/ folder

**Comment by GearNut** [comment]
Moved .step files into a cad folder in the Stls folder

**Comment by newphreak** [comment]
printer_mods/Gearnut/V0_Misumi_LED_Mount/CAD
printer_mods/Gearnut/V0_Misumi_LED_Mount/STL
We'll get there soon

**Comment by GearNut** [comment]
.step files moved

**Comment by newphreak** [comment]
Still not in it's own subfolder of the mod like shown above

**Comment by GearNut** [comment]
The folder should be in the same spot as the folder for images and stl right? thats where I see it on my end. 

printer_mods/Gearnut/V0_misumi_LED_Mount/CAD
printer_mods/Gearnut/V0_misumi_LED_Mount/Images
printer_mods/Gearnut/V0_misumi_LED_Mount/STL

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Assembly is fairly straight forward. Solder directly to the LED strip or solder on a JST connector to the end and slide the strip inside. If your leds come with a clear silicone layer over the top you may need to peel it off the strip or file down the insides of the led tracks, the bare pcb backed LEDs should fit in snugly. 
+
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_1.jpg)
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_2.jpg)
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_3.jpg)
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_4.jpg)
+
+You will need 5 per side and standard voron print settings are probably overkill for these but its what I used. For install on a stock V0 simply remove the rail and slide inside. Fitment for me was tight and the rails did shave a little bit off the mounting points. This was intentional to make it as tight as possible.
```
Comment (newphreak) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:17]: extrusion?

**Review by newphreak — CHANGES_REQUESTED** [review]
Otherwise it looks good now, sorry for the tedious process. You're doing great.

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md (hunk 1 above)
Comment (GearNut) [reply printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md]: changed rails to extrustions. Thank you didnt catch that one.

**Comment by newphreak** [comment]
[X ] The PR modifies the top-level README.md of the category folder adding the
contribution to the table. Read the top part of the file for instructions on how
to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
to fill out the compatibility matrix to indicate which versions of the Voron printer
the submission is compatible with.

Oh, you need to add your name and mod in the top level README also.

**Comment by GearNut** [comment]
Added. That was the only part of the PR I was confused on, just got lost in so much code but Im in the table

**Review by newphreak — APPROVED** [review]
Looks good to me. Good job 👍

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+V0 Misumi Led mount
+
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Cover.jpg)
+
+These were modified from the Misumi Led clips(https://www.teamfdm.com/files/file/338-misumi-led-corners/) as I like the design with them in my V2.4 They should fit most WS2812b Led strips you can find online though If it has the silicone cover on them you may need to peel them off or file down the inside track a touch.
```
Comment (cbc02009) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:5]: Is this mod available in this github repo? we'd prefer you link to it from here instead.

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+I did include both the extended version I have tested and use personally as well as the shorter version If youre LED strip as more LEDs per meter than mine did, while it will print and install just fine they are as of now untested.
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Short_Tall.png)
```
Comment (cbc02009) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:8]: relative link

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Assembly is fairly straight forward. Solder directly to the LED strip or solder on a JST connector to the end and slide the strip inside. If your leds come with a clear silicone layer over the top you may need to peel it off the strip or file down the insides of the led tracks, the bare pcb backed LEDs should fit in snugly. 
+
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_1.jpg)
```
Comment (cbc02009) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:12]: relative links here too.

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_2.jpg)
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_3.jpg)
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Install_4.jpg)
+
+You will need 5 per side and standard voron print settings are probably overkill for these but its what I used. For install on a stock V0 simply remove the extrusion and slide inside. Fitment for me was tight and the extrusion did shave a little bit off the mounting points. This was intentional to make it as tight as possible.
+
+These did go through several variations and attempts with Tinkercad to get right but I hope you enjoy them!
+
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Version_History.png)
```
Comment (cbc02009) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:21]: and here.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Please fix the minor readme issues and this should be good to go.

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md (hunk 2 above)
Comment (GearNut) [reply printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:6]: Unfortunately I dont think it is. I Have looked around and cant find it and the teamfdm link has a git hub link on it but the link is dead and ramalama2 has it in his depreciated folder on his guthub. Team fdm was the most user friendly site I could find to link.

**Comment by GearNut** [comment]
Well that took some figuring out to do but all images should be relative links now and replied directly but ill state it here too. long and short I think at one point the model I remixed was on the voron github but not any more

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md (hunk 2 above)
Comment (newphreak) [reply printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:6]: Isn't it this one? https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/eddie/LED_Bar_Clip

Comment (GearNut) [reply printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:6]: So the first link you used was the correct one and the file I started with. just used a basic 1515 stl file from somewhere(i think thingiverse) for the 1515 clip and just modelled the two together with some tweaks for fitment. 

so ill update it to the first one as thats still the stl I use in my V2 and VT

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+V0 Misumi Led mount
+
+![Alt_text](./Images/Cover.jpg)
+
+
+These were modified from Eddie's Misumi Led clips (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/eddie/LED_Bar_Clip) as I like the design with them in my V2.4 and VT. They should fit most WS2812b Led strips you can find online though If it has the silicone cover on them you may need to peel them off or file down the inside track a touch.
```
Comment (cbc02009) [inline printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md:6]: Can you make this a relative link too? (`../../printer_mods/eddie/LED_Bar_Clip`)

**Review by cbc02009 — APPROVED** [review]

File: printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md (hunk 6 above)
Comment (GearNut) [reply printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md]: Well figured that one out faster. its a relative link now as well

**Comment by cbc02009** [comment]
Looks like there's a typo in one of the links in the readme that needs fixing.

**Comment by GearNut** [comment]
Well that was odd. Had to remove the printer_mods from the relative link to make it work

(../../

printer_mods

/eddie/LED_Bar_Clip)

**Comment by GearNut** [comment]
Also tried to do this on my phone so If I made a mess of things let me know and I'll try to fix them later.

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**